src/components/sbb-expansion-panel/sbb-expansion-panel.scss

```diff
@@ -0,0 +1,108 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-expansion-panel-animation-duration: var(--sbb-animation-duration-4x);
+  --sbb-expansion-panel-background-color: var(--sbb-color-white-default);
+  --sbb-expansion-panel-content-visibility: hidden;
+  --sbb-expansion-panel-content-max-height: 0;
+  --sbb-expansion-panel-content-opacity: 0;
+  --sbb-expansion-panel-content-transition: max-height var(--sbb-expansion-panel-animation-duration)
+      var(--sbb-animation-easing),
+    padding var(--sbb-expansion-panel-animation-duration) var(--sbb-animation-easing),
+    opacity var(--sbb-expansion-panel-animation-duration) var(--sbb-animation-easing);
+  --sbb-expansion-panel-border-color: var(--sbb-color-cloud-default);
+  --sbb-expansion-panel-border: var(--sbb-border-width-1x) solid
+    var(--sbb-expansion-panel-border-color);
+  --sbb-expansion-panel-border-top: 0;
+  --sbb-expansion-panel-top-right-radius: 0;
+  --sbb-expansion-panel-top-left-radius: 0;
+  --sbb-expansion-panel-bottom-right-radius: 0;
+  --sbb-expansion-panel-bottom-left-radius: 0;
+}
+
+:host([disabled]:not([disabled='false'])) {
+  @include sbb.if-forced-colors {
+    --sbb-expansion-panel-border-color: GrayText;
+  }
+}
+
+:host([expanded]) {
+  --sbb-expansion-panel-content-visibility: visible;
+  --sbb-expansion-panel-content-max-height: calc(
+    var(--sbb-expansion-panel-content-height) + var(--sbb-spacing-responsive-xs) +
+      var(--sbb-spacing-responsive-xs)
+  );
+  --sbb-expansion-panel-content-opacity: 1;
+  --sbb-expansion-panel-content-transition: max-height var(--sbb-expansion-panel-animation-duration)
+      var(--sbb-animation-easing),
+    padding var(--sbb-expansion-panel-animation-duration) var(--sbb-animation-easing),
+    opacity var(--sbb-expansion-panel-animation-duration)
+      var(--sbb-expansion-panel-animation-duration) var(--sbb-animation-easing);
+}
+
+:host(:first-child) {
+  --sbb-expansion-panel-border-top: var(--sbb-border-width-1x) solid
+    var(--sbb-expansion-panel-border-color);
+  --sbb-expansion-panel-top-right-radius: var(--sbb-border-radius-4x);
```

Comment:
border-radius should be a variable as used at multiple locations

Reply:
See 4d1f901